Housey-pricey.js

```diff
@@ -0,0 +1,18 @@
+"use strict";
+let volumeInMeters = 8 * 10 * 10;
+let gardenSizeInM2 = 100;
+let housePrice = volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300;
+const housePeter = 2500000;
+let priceComparison = housePeter > housePrice ? `over` : `under`;
+let buyer = "Peter";
+
+console.log(`${buyer} is ${priceComparison}paying for the house`);
```

Comment:
missing space between `${priceComparison}` and `paying`, but it's fine lol

Reply:
overpay and underpay are supposed to be 1 word cer haha